tests/test-user.js

```diff
@@ -571,6 +573,52 @@ describe('User', () => {
       });
     });
 
+    it('should log error and return undefined when GET OVH returns 500 error', (done) => {
```

Comment:
Vu le nom du test j'imagine qu'il faut aussi vérifier que dans la console on a bien le message d'erreur ? Avec sinon vous pouvez mocker la console ([petit example SO ici](https://stackoverflow.com/questions/29802117/sinon-spy-on-console-log-call-not-registered)) et verifier que console.**error** est appelé avec ce dont vous vous attendez